src/asynk/async_queue.rs

```diff
@@ -912,6 +878,27 @@ mod async_queue_tests {
         test.transaction.rollback().await.unwrap();
     }
 
+    #[tokio::test]
+    async fn schedule_task_test() {
+        let pool = pool().await;
+        let mut connection = pool.get().await.unwrap();
+        let transaction = connection.transaction().await.unwrap();
+
+        let mut test = AsyncQueueTest::builder().transaction(transaction).build();
+
+        let task = test
+            .schedule_task(&AsyncTaskSchedule { number: 1 })
```

Comment:
please check scheduled_at field of this task

Reply:
[Comments adressed](https://github.com/ayrat555/fang/pull/71/commits/9c6c34faf584651f259dec069083509f78ce5934)